tests/models/emerge/src/example.py

```diff
@@ -0,0 +1,232 @@
+import casadi as ca
```

Comment:
@FJanssen-TNO  you mentioned this will still be updated/modified. My suggestion, some commented/TODO or something to point out that this file is wip (work in progress) and the code & commented still has to be finalized. Otherwise someone else might look at this code without clear commenting or wondering why there is pieces of code that is commented out

Reply:
added